### DISCUSSION TITLE
Using Rclone mount inside a container doesn't show files in the directory

### DISCUSSION THREAD

**Discussion by Sybion1411** [containers/podman#23895] [Q&A] [answered]
Hello,
I have a problem mounting Google Drive using rclone in a container - it doesn't show any files, but the command `rclone ls piokug96:` (piokug96 is the name of my remote) returns a full list of the files and the sync/copy functions work fine. Also on the [rclone forum](https://forum.rclone.org/t/rclone-not-working-on-steam-deck-using-distrobox/47503) I was told that rclone works properly.
I found the [discussion](https://github.com/containers/podman/issues/12879) and used the command
`podman unshare mount --bind --make-shared /home/deck/ /home/deck/`
but it didn't help (I mount directory /home/deck/GDrive-piokug96, so the parent is /home/deck/, right?).

Do you know a possible solution or have any knowledge on how to fix this?

**Comment by rhatdan** [comment]
Do you have an example of the podman command you are using to launch the container?

**Reply by Sybion1411** [reply, marked as answer]
I used BoxBuddy and as far as I know Podman is the host system for it, so I didn't use any podman command.
Actually I did another reboot and rclone works perfectly fine now. I don't know what was the issue here.
Through nix packages rclone worked also fine - I used it as workaround.
Thank you for your answer :)
